PanKanapka/bulka/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+
+from django.db import models
+
+class Alergeny(models.Model):
+    Ale_Id = models.IntegerField(primary_key = True)
```

Comment:
W parametrach kluczowych `funkcja(a=b)` nie powinno być spacji.
To może wydawać się nieistotną uwagą, ale serio zwracają na to uwagę w robocie, więc dla czystości kodu warto by to zmienić.

Reply:
Ok, dostosuję się do Twojego "coding style" natomiast moim zdaniem powinno być albo tak jak jest:
Ale_Id = models.IntegerField(primary_key = True)
albo
Ale_Id=models.IntegerField(primary_key=True)
bo operacja przypisania jest operacją przypisania nie ważne gdzie i rozwiązanie
Ale_Id = models.IntegerField(primary_key=True)
jest dość dziwaczne.

Reply:
Tylko gwoli wyjaśnienia - to jest standard narzucany przez pylinta i PEP8; dla pewności zrobiłam test i w obydwu przypadkach rzuca warningiem:
`No space allowed around keywoard argument assignment`
oraz
`Exactly one space required around assignment`.